include/openssl/types.h

```diff
@@ -70,15 +82,6 @@ typedef struct ASN1_ITEM_st ASN1_ITEM;
 typedef struct asn1_pctx_st ASN1_PCTX;
 typedef struct asn1_sctx_st ASN1_SCTX;
 
-# ifdef _WIN32
-#  undef X509_NAME
-#  undef X509_EXTENSIONS
-#  undef PKCS7_ISSUER_AND_SERIAL
```

Comment:
In prototyping the modified wincrypt.h, I could not find any reference to `PKCS7_ISSUER_AND_SERIAL` in Windows headers - so I removed the undef in this PR.